README.md

```diff
@@ -186,10 +186,10 @@ Here is a rundown of other arguments for your docker-compose / docker run.
 ### Installing on Ubuntu or Fedora
 
 Modern releases of Ubuntu (17.10+) and Fedora (33+) include [`systemd-resolved`](http://manpages.ubuntu.com/manpages/bionic/man8/systemd-resolved.service.8.html) which is configured by default to implement a caching DNS stub resolver. This will prevent pi-hole from listening on port 53.
-The stub resolver should be disabled with: `sudo sed -r -i.orig 's/#?DNSStubListener=yes/DNSStubListener=no/g' /etc/systemd/resolved.conf`.
+The stub resolver should be disabled with: `sudo sh -c 'mkdir -p /etc/systemd/resolved.conf.d && printf "[Resolve]\nDNSStubListener=no\n" | tee /etc/systemd/resolved.conf.d/no-stub.conf'`.
```

Comment:
I like this a lot, as it does what we also do on bare metal installations

https://github.com/pi-hole/pi-hole/blob/3977a312f01687bcc649365297c6a0a3ba402317/automated%20install/basic-install.sh#L1300-L1305

We probably need to reload `systemd-resolved` also to take the new configuration